### PR TITLE
Editing while reading

### PR DESCRIPTION
**Pull request by jean**
Assuming I'm in the `demo` directory, unless I append '.' to my path,
import fails.

This feels rather hacky, is there a better way?

### CODE REVIEW

**Comment by michwill** [comment]
That is very strange. I cannot reproduce this (import doesn't fail in ipython; using python2.7 and 3.5). I am testing in virtualenv, in Linux, but my thinking usually was that python can do 'import X' when X.py is in current directory.

**Comment by jean** [comment]
Yes, that's what I expect too. I'll check some more.

J

**Comment by jean** [comment]
@michwill the `sys.path` issue only shows up when trying to follow the example in the `zerodb-manage console`

File: docs/source/index.rst [hunk 1]
```diff
@@ -15,47 +15,54 @@ Overview
...
 
 Installation
 ============
+
 We supply ZeroDB as a Python Package ``zerodb-0.97``, installable via ``pip``.
 
+There may also be some system dependencies; on Debian, this will get the
+required packages::
+
+    $ sudo apt-get install python-pip python-dev libssl-dev libzmq-dev libffi-dev git python-virtualenv
```
Comment (michwill) [inline docs/source/index.rst]: Huh. Why is zmq required? Somehow fails to build without it?

Comment (jean) [inline docs/source/index.rst]: Yeah, from #47:

```
vagrant@vagrant-ubuntu-trusty-64:~$ zerodb-manage initdb
[...]
ImportError: No module named ipython_genutils
[...]
vagrant@vagrant-ubuntu-trusty-64:~$ pip install --user jupyter
[...]
    build/temp.linux-x86_64-2.7/scratch/vers.c:4:17: fatal error: zmq.h: No such file or directory
```

**Pull request merged**